crates/codegen/parser/runtime/src/kinds.rs

```diff
@@ -36,6 +36,12 @@ pub enum RuleKind {
     XXX,
 }
 
+impl RuleKind {
+    pub fn is_trivia(&self) -> bool {
+        todo!()
```

Comment:
This is expanded - maybe this should be `unreachable!("Expanded by the template")`?

Reply:
Changed in 31dd8e3d0cbe5f723d6a9fd176e23142cc861ac4